packages/frontend/core/src/components/affine/setting-modal/general-setting/about/style.css.ts

```diff
@@ -43,3 +43,37 @@ globalStyle(`${communityItem} p`, {
   fontSize: 'var(--affine-font-xs)',
   textAlign: 'center',
 });
+
+export const checkUpdateDesc = style({
+  color: 'var(--affine-text-secondary-color)',
+  fontSize: 'var(--affine-font-xs)',
+  display: 'inline-flex',
+  alignItems: 'center',
+  justifyContent: 'flex-start',
+  selectors: {
+    '&.active': {
+      color: 'var(--affine-text-emphasis-color)',
+    },
+    '&.error': {
+      color: 'var(--affine-error-color)',
+    },
+  },
+});
+
+globalStyle(`${checkUpdateDesc} svg`, {
+  marginRight: '4px',
```

Comment:
maybe using column gap in checkUpdateDesc?

Reply:
I want to use gap attributes as little as possible

Reply:
can you elaborate?